superduperdb/base/build.py

```diff
@@ -74,7 +76,21 @@ def build(uri, mapping):
         logging.error("Error initializing to DataBackend Client:", str(e))
         sys.exit(1)
 
-    # Build a Datalayer object with the specified components.
+    # Connect to compute backend.
+    # ------------------------------
+    compute = None
+    if cfg.cluster.distributed:
```

Comment:
rather than this, it should be a Factory class or method which creates a ComputeBackend class based on distributed true or not